samples/kubernetes/README.md

```diff
@@ -0,0 +1,67 @@
+# Running in Kubernetes
+
+*IMPORTANT:* This configuration is not suitable for production scenarios. This sample is provided as an example of how the Fhir Server project can be run on Kubernetes. Please note by default the SQL server connection is not encrypted and authentication on the FHIR Server has been disabled.
+
+If you do not have a Kubernetes cluster and want to deploy Kubernetes on Azure documentation can be found here: [https://docs.microsoft.com/en-us/azure/aks/](https://docs.microsoft.com/en-us/azure/aks/)
+
+## Optional: Deploying SQL Server on Kubernetes
+
+The FHIR Server can be connected to a SQL server intance external to Kubernetes, for example Azure SQL Database. However if you wish to run SQL Server on Kubernetes follow the steps below:
+
+### Create secret for SA password
+
+Create a Kubernetes secret to store the SQL Server SA password, `<SA_PASSWORD>`:
+
+```bash
+kubectl create secret generic sql-settings --from-literal=password='<SA_PASSWORD>'
+```
+
+### Deploy SQL Server
+
+The example SQL server deployment creates a Kubernetes deployment, load balancer of type `ClusterIP`, and a persistant volume claim is created with aa storage class of `volume.beta.kubernetes.io/storage-class: managed-premium`. If you are not running Azure Kubernetes Service you will need to adjust the storage class to one that exists on your cluster.
```

Comment:
nit: "aa storage class..." -> "a storage class"

Reply:
Taken k8s out. Will fix in new PR.